server/src/main/java/io/crate/metadata/Schemas.java

```diff
@@ -264,6 +267,17 @@ private static String relationName(QualifiedName ident) {
         return parts.get(parts.size() - 1);
     }
 
+    /// @throws [IndexNotFoundException]
+    /// @throws [RelationUnknown]
+    public DocTableInfo getTableInfo(Index index) {
```

Comment:
Looks like we already use the same logic in other places, found two places:`TransportShardUpsertAction.processRequestItems` and`TransportShardUpsertAction.processRequestItemsOnReplica`). Such we could replace it with this method call.